File: FastMines_core/src/fmg/core/app/model/MosaicBackupData.java
package fmg.core.app.model;

import java.util.List;

import fmg.common.geom.Matrisize;
import fmg.core.mosaic.cells.BaseCell;
import fmg.core.types.EMosaic;

/** Mosaic controller backup data */
public class MosaicBackupData {

    public EMosaic mosaicType;

    public Matrisize sizeField;

    public List<BaseCell.StateCell> cellStates;

    public double area;

    public int clickCount;

}

File: FastMines_core/src/fmg/core/app/model/MosaicBackupData.cs
using System.Collections.Generic;
using Fmg.Common.Geom;
using Fmg.Core.Types;
using Fmg.Core.Mosaic.Cells;

namespace Fmg.Core.App.Model {

    /// <summary> Mosaic controller backup data </summary>
    public class MosaicBackupData {

        public EMosaic MosaicType { get; set; }

        public Matrisize SizeField { get; set; }

        public IList<BaseCell.StateCell> CellStates { get; set; }

        public double Area { get; set; }

        public int ClickCount { get; set; }

    }

}